### ISSUE TITLE
fix ci job

### ISSUE THREAD

**Issue by merikan** [spaceship-prompt/spaceship-flutter#1]
5 out of 6 tests fails in the ci job
```
Spaceship version: 4.3.3
test_flutter_no_files
test_flutter_upsearch_is_not_flutter_project
test_flutter_upsearch_is_flutter_project
ASSERT:should render when file pubspec.yaml has flutter dependency expected:<%{%B%}via %{%b%}%{%B%F{blue}%}💙 v3.3.3 #stable%{%b%f%}%{%B%} %{%b%}> but was:<>
ASSERT:should render when file pubspec.yml has flutter dependency expected:<%{%B%}via %{%b%}%{%B%F{blue}%}💙 v3.3.3 #stable%{%b%f%}%{%B%} %{%b%}> but was:<>
test_flutter_show_no_channel
ASSERT:should render when file pubspec.yaml has flutter dependency expected:<%{%B%}via %{%b%}%{%B%F{blue}%}💙 v3.3.3%{%b%f%}%{%B%} %{%b%}> but was:<>
test_flutter_with_beta_channel
ASSERT:should render with beta channel expected:<%{%B%}via %{%b%}%{%B%F{blue}%}💙 v3.4.0-34.1.pre #beta%{%b%f%}%{%B%} %{%b%}> but was:<>
test_flutter_has_updates
ASSERT:should render when flutter has updates expected:<%{%B%}via %{%b%}%{%B%F{blue}%}💙 v3.3.3 #stable%{%b%f%}%{%B%} %{%b%}> but was:<>

Ran 6 tests.

FAILED (failures=[5](https://github.com/spaceship-prompt/spaceship-flutter/actions/runs/3217172793/jobs/5259840054#step:5:6))
Error: Process completed with exit code 1.
```

**Comment by merikan** [comment]
The image used in ci jobs is "ghcr.io/spaceship-prompt/spaceship-prompt" and the image does not contain any of the packages used in the function 'spaceship::datafile' (yq,ruby,python3) which is used to read a structured file

**Comment by merikan** [comment]
fix in 2d87258b327661aca12437d75a5ef118702ff388
closing issue
